src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -692,3 +588,112 @@ def update_connectedk8s(cmd, instance, tags=None):
     with cmd.update_context(instance) as c:
         c.set_param('tags', tags)
     return instance
+
+# pylint:disable=unused-argument
+# pylint: disable=too-many-locals
+# pylint: disable=too-many-branches
+# pylint: disable=too-many-statements
+# pylint: disable=line-too-long
+
+
+def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="",
+                  kube_config=None, kube_context=None, no_wait=False, tags=None):
```

Comment:
I think we can remove tags parameter from here since we are not using it anywhere.